src/python/tests/test_libs/untrusted_runner_helpers.py

```diff
@@ -196,6 +197,10 @@ def setUp(self):
 
     environment.set_value('GSUTIL_PATH', os.path.dirname(_which('gsutil')))
 
+    test_utils.setup_pubsub('test-clusterfuzz')
```

Comment:
nit: can we use utils.default_project_name to avoid hardcoding 'test-clusterfuzz' in these two statements.